.changeset/gorgeous-eggs-brake.md

```diff
@@ -0,0 +1,5 @@
+---
+'@eth-optimism/l2geth-exporter': patch
+---
+
+Bump to go-ethereum v1.10.16
```

Comment:
Is this a standard version we're using? Any reason not to go to 1.17/18?

Reply:
It's the minimum version we're using everywhere else. 1.18 is latest, but contains the new generics code and we should wait a bit before adopting it.